homeassistant/components/pvpc_hourly_pricing/const.py

```diff
@@ -0,0 +1,11 @@
+"""Constant values for pvpc_hourly_pricing."""
+from aiopvpc import TARIFFS
+
+DOMAIN = "pvpc_hourly_pricing"
+PLATFORM = "sensor"
+ATTR_TARIFF = "tariff"
+DEFAULT_NAME = "PVPC"
+DEFAULT_TARIFF = TARIFFS[1]
+DEFAULT_TIMEOUT = 5
+
+UNIQUE_ID_MASK = "pvpc_price_sensor_tariff_{0}"
```

Comment:
Unique ids are unique to the DOMAIN.  You can just use tariff here since its unique.

Reply:
I just wrapped the tariff with a more descriptive string, but I'll remove this and use just the tariff